src/program/lwaftr/tests/propbased/selftest.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+SKIPPED_CODE=43
+
+if [ -z $SNABB_PCI0 ]; then exit $SKIPPED_CODE; fi
+
+./snabb quickcheck program.lwaftr.tests.propbased.prop_nocrash $SNABB_PCI0
```

Comment:
Ok for now, though ideally I'd like to be able to run these tests without a real NIC at some point, so I can just have my personal build server hammer away at it. Not a blocker for merging.